packages/react-code-editor/src/components/CodeEditor/CodeEditor.tsx

```diff
@@ -551,6 +568,16 @@ export class CodeEditor extends React.Component<CodeEditorProps, CodeEditorState
                     <CodeEditorContext.Provider value={{ code: value }}>{customControls}</CodeEditorContext.Provider>
                   )}
                 </div>
+              }
+              {<div className={css(styles.codeEditorHeaderMain)}>{headerMainContent}</div>}
+              {!!shortcutsPopoverProps.bodyContent && (
+                <div className="pf-c-code-editor__keyboard-shortcuts">
+                  <Popover {...shortcutsPopoverProps}>
+                    <Button variant={ButtonVariant.link}>
```

Comment:
This will wrap the icon in an element and apply the appropriate layout and spacing.

```suggestion
                    <Button variant={ButtonVariant.link} icon={<HelpIcon />}>
```